src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -108,6 +108,21 @@
         return emails;
     }
 
+    /**
+     * Generates the feedback session reminder emails for the given {@code student}.
+     */
+    public EmailWrapper generateFeedbackSessionStudentReminderEmail(
```

Comment:
Is this the same as the method above? `generateFeedbackSessionReminderEmails `?

Reply:
No. This is only for a particular student. The method above generates for all students and instructors.

Reply:
But the code is duplicated (almost the same?); By right, `generateFeedbackSessionReminderEmails` should be able to do `generateFeedbackSessionReminderEmails` can do.

Reply:
Yes `generateFeedbackSessionReminderEmails` can do what `generateFeedbackSessionStudentReminderEmail` does. But I didn't want to generate emails for everyone even though we only need one of them. Seems quite wasteful.

I think it's more like `generateFeedbackSessionReminderEmails` can call `generateFeedbackSessionStudentReminderEmail`? I could refactor that.